CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-### Changes in 0.24
+### master
+
+- Fixed: Security bug: HSTS header is sent over HTTP and not HTTPS [#1267](https://github.com/luckyframework/lucky/issues/1267)
+
+### v0.24.0 (2020-09-05)
```

Comment:
Don't worry about adding in CHANGELOG stuff. That's all taken care of when we do a release.

Reply:
Alrighty, I'll remove the commit

Reply:
Probably best take that out of the contributing guide in the readme then. And make there only be one list of contributing steps (`CONTRIBUTING.md` vs `README.md`) =P